mls-rs-core/src/crypto.rs

```diff
@@ -68,6 +70,7 @@ impl AsRef<[u8]> for HpkePublicKey {
     all(feature = "ffi", not(test)),
     safer_ffi_gen::ffi_type(clone, opaque)
 )]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 pub struct HpkeSecretKey(#[mls_codec(with = "mls_rs_codec::byte_vec")] Vec<u8>);
```

Comment:
I think before we had a serde with here that would base64 the byte arrays, otherwise you get really ugly JSON with each byte as a number

Reply:
I thought about it but didn't know how to integrate with `cfg_attr` without causing a mess. Is there a way?

Reply:
Should be, let me take a look on Monday